us_state_abbrev.py

```diff
@@ -57,4 +59,9 @@
     "Wyoming": "WY",
 }
 
-us_state_name_by_abbrev = dict(map(reversed, us_state_abbrev_by_name.items()))
+
+def swap(a: Tuple[str, str]) -> Tuple[str, str]:
+    return (a[1], a[0])
+
+
+us_state_name_by_abbrev: Mapping[str, str] = dict(map(swap, us_state_abbrev_by_name.items()))
```

Comment:
mypy didn't seem to be smart enough to understand `reversed` when applied to `Tuple[str, str]`.